src/shared/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Box, CircularProgress } from '@mui/material';
+import { LoaderType } from './constants/constants';
+import './Loader.scss';
+
+interface LoaderProps {
+  type: string;
+}
+
+export const Loader: React.FC<LoaderProps> = ({ type }: LoaderProps) => {
+  const isLarge: boolean = type === LoaderType.LOADER_TYPE_LARGE;
```

Comment:
move isLarge to props and make it true by default

Reply:
Moved to default